File System Access API landing page: Fix examples and improve language. (#24923)
mdn/content: files/en-us/web/api/file_system_access_api/index.md

### Accessing files

The below code allows the user to choose a file from the file picker.

```js
async function getFile() {

>   // open file picker (and destructure the result to only obtain one handle)

Reviewer: Nit
```suggestion
  // Open file picker and destructure the result the first handle
```

};

async function getTheFile() {

>   // open file picker (and destructure the result to only obtain one handle)

Reviewer: ```suggestion
  // Open file picker and destructure the result the first handle
```
